src/nvim/vim.h

```diff
@@ -316,8 +316,60 @@ enum { FOLD_TEXT_LEN = 51 };  //!< buffer size for get_foldtext()
 #define LOWEST_WIN_ID 1000
 
 // BSD is supposed to cover FreeBSD and similar systems.
-#if (defined(BSD) || defined(__FreeBSD_kernel__)) && defined(S_ISCHR)
-# define OPEN_CHR_FILES
+#if (defined(BSD) || defined(__FreeBSD_kernel__)) \
+    && (defined(S_ISCHR) || defined(S_IFCHR))
+#define OPEN_CHR_FILES
```

Comment:
indent by 1 space.

Reply:
I got an error with make lint when I set the indent to 1, so I set the indent to 2, as shown below.

```c
#if (defined(BSD) || defined(__FreeBSD_kernel__)) \
  && (defined(S_ISCHR) || defined(S_IFCHR))
#define OPEN_CHR_FILES
```

Reply:
That's not what I meant.
Preprocessor directives can be indented after the `#` character. Vim uses this to inform the developer of the nesting of directives which can spread across hundreds of lines of code.

Reply:
Wait, this line wasn't modified in the Vim patch.

Reply:
Thank you for telling us about Preprocessor directives. It was a good learning experience.

Reply:
I thought it had been changed [here](https://github.com/vim/vim/commit/d569bb029983cff947dce704e6f830276204c13f#diff-0f9fae9859e0cdbca395251787c939a8bf5ba976ffa1f87615936adc193f1be4R2510-R2512), am I wrong?
If I made a mistake, I will correct it.

Reply:
Condition was modified, not `# define`.

Reply:
So you meant `# define`.
Sorry for my slow understanding.

We'll fix it.